apps.yaml

```diff
@@ -109,3 +109,19 @@ prisma:
   categories:
      - data-analysis
      - utilities
+clease-gui:
+  metadata:
+    title: CLEASE GUI
+    description: |
+      A notebook-based graphical user interface for the Cluster Expansion code CLEASE.
+      The app is only a GUI, and requires a working version of the CLEASE code.
+      For more information on CLEASE, please refer to the CLEASE documentation (https://clease.readthedocs.io/).
+    authors: Alexander S. Tygesen, Jin Hyun Chang
+    external_url: https://gitlab.com/computationalmaterials/clease-gui
+    documentation_url: https://clease-gui.readthedocs.io
+    logo: https://gitlab.com/computationalmaterials/clease-gui/-/raw/master/docs/source/_static/clease_logo_no_text.svg
```

Comment:
The logo file would need to be a file that can be directly displayed in a browser. Most browsers do not natively support embedded `.svg` files. Could you replace this with a rendered version of the logo, please? The .png or .jpg formats would be preferred.

Reply:
People need to get proper browsers then :-) I'll make a rendered version.

Reply:
It's pointing to a `.png` version now. Just had to verify the resolution was acceptable.